operator/apis/platform/v1alpha1/common_types.go

```diff
@@ -324,3 +324,11 @@ type OpenShiftMonitoring struct {
 func (m *GlobalMonitoring) IsOpenShiftMonitoringDisabled() bool {
 	return m != nil && m.OpenShiftMonitoring != nil && !m.OpenShiftMonitoring.Enabled
 }
+
+// GlobalSystemSpec defines settings related to Helm chart system parameters. The corresponding Helm flag
+// lives in the global scoope `.system`.
+type GlobalSystemSpec struct {
+	//+kubebuilder:default=true
+	//+operator-sdk:csv:customresourcedefinitions:type=spec,order=1,xDescriptors={"urn:alm:descriptor:com.tectonic.ui:booleanSwitch"}
+	EnableNetworkPolicies *bool `json:"enableNetworkPolicies,omitempty"`
```

Comment:
Just wondering, would it make sense in the future to not only have this a boolean, but instead choose from a certain set of pre-defined network policy setups? Currently we would provide the options "none" and "default" or something alone these lines?
I haven't thought about this deep, hence this is more a general question, because going from a boolean to an enum is a pattern occuring from time to time.